api/swagger.yaml

```diff
@@ -3341,7 +3358,7 @@ components:
               type: string
               format: date-time
               description: This constraint applies from the given start date/time.
-              example: '2012-01-01T00:00:00Z'
+              example: '2012-01-01T00:00Z'
```

Comment:
Waarom heb je hier de seconden weggehaald? De standaard notatie is mét seconden.

Reply:
In eerste instantie omdat dit meer consistentie oplevert in hoe we date times lezen en schrijven, maar ook omdat het toegestaan is door de ISO standaard. Het kan best zijn dat een tijdstip met een precisie in minuten (dus zonder seconden/milliseconden) wordt weergegeven. Of dat ie zelfs een precisie in uren heeft.

Bron: https://en.wikipedia.org/wiki/ISO_8601#Times

> Either the seconds, or the minutes and seconds, may be omitted from the basic or extended time formats for greater brevity but decreased accuracy: [hh]:[mm], [hh][mm] and [hh] are the resulting reduced accuracy time formats.

Reply:
Good thinking then!